### PR TITLE
Trial expired message

### PR DESCRIPTION
**Pull request by jorgesierra**
Screen Shot 2021-03-03 at 18 55 09

### CODE REVIEW

File: src/ui/components/ext-header/ext-header.tsx [hunk 1]
```diff
@@ -46,9 +46,10 @@ const ExtHeader = (props: IExtHeader) => {
     return (
         <Header>
             <Row>
-                <Col span={4}>
+                <Col span={10}>
                     <a href="https://app.tealhq.com/job-tracker" target="_blank">
                         <img src={chrome.runtime.getURL('images/teal_logo_32.svg')} />
+                        <span className="pro-label">PRO</span>
```
Comment (ohaicristina) [inline src/ui/components/ext-header/ext-header.tsx:52]: I would pop this out of the `<a>`

**Review by ohaicristina — COMMENTED** [review]
Code looks good with one small comment! I tried testing this out locally but I couldn't get it to work 🤔  (seems like the code that needs to be uncommented in `../login.tsx` isn't the right one?) 
let me know how to trigger it so i can take a look just to double check and this should be good to go!

**Comment by jorgesierra** [comment]
@ohaicristina changing the jwt payload in `login.tsx`should work. It is required to log out and log in again as the jwt is decoded only after signing in. Steps are:
1. Change code for testing
2. `npx webpack`
3. Load new version into the browser
4. Log out if necessary and log in again.

While confirming these steps work I found that I missed checking for the free plan 🤦 . I updated the pr, now we check for plan type `Trial` and `Free`, and in the case of having a trial plan, we check that the date is expired.

**Review by ohaicristina — APPROVED** [review]
@jorgesierra - ahh i realized that earlier i had been clicking on the live version of the job tracker extension instead of the built one uploaded from `dist`. we're good!

works perfectly! let's merge it up!

**Pull request merged**